docs/user/pixel_observations.md

```diff
@@ -55,13 +55,13 @@ working with pixel observations. You may use multiple wrappers simultaneously,
 though the order in which you use them matters. See the pixel-specific wrappers
 section for more details on this.
 
-### GarageEnv
+### GymEnv
```

Comment:
Don't all instances of Environment accept `is_image`?

Reply:
No? For example `PointEnv` doesn't accept `is_image`.